### PR TITLE
chore: add note about NFS

### PR DESCRIPTION
**Pull request by bsmth**
This PR adds a warning about using networked filesystems and that QuestDB officially supports **EXT4** or **XTS** or any filesystem that supports [mmap](https://man7.org/linux/man-pages/man2/mmap.2.html) to the capacity planning doc.

### CODE REVIEW

**Review by bluestreak01 — APPROVED** [review]

File: docs/operations/capacity-planning.md [hunk 1]
```diff
@@ -18,10 +16,20 @@ settings are configured in QuestDB by either a `server.conf` configuration file
...
+of data and filesystem considerations.
+
+:::caution
 
-The following section describes aspects to be considered regarding the storage
-of data.
+- QuestDB officially supports **EXT4** or **XTS** or any filesystem that
+  supports [mmap](https://man7.org/linux/man-pages/man2/mmap.2.html).
+
+- Users **can't use NFS or a similar networked filesystem** directly with a
```
Comment (puzpuzpuz) [inline docs/operations/capacity-planning.md:29]: `networked filesystem` should be `distributed file system`

Comment (bsmth) [reply docs/operations/capacity-planning.md:29]: I see the PR's merged, I'll follow-up with a typo fix 👍🏻

Comment (puzpuzpuz) [reply docs/operations/capacity-planning.md:29]: Thanks!

Comment (bsmth) [reply docs/operations/capacity-planning.md:29]: done in https://github.com/questdb/questdb.io/pull/662

**Pull request merged**